src/pages/api/multipleProductValidity.ts

```diff
@@ -125,7 +124,7 @@ export default async (req: NextApiRequest, res: NextApiResponse): Promise<void>
             throw new Error('Could not extract the user email address from their ID token');
         }
 
-        const multipleProductPeriod: DecisionData = {
+        const multipleProductPeriod = {
```

Comment:
Should we be replacing this type we've removed?

Reply:
This code will be removed entirely by the work being done on 535, so no need to annotate with a type at this stage.